File: API_AppBanNoiThat/model/gioHang.js
const mongoose = require('mongoose')

const GioHangsSchema = new mongoose.Schema({
    nguoi_dung_id: {
        type: String,
        ref: 'nguoiDungs'
    },
    // tong_gia: {
    //     type: Number
    // },
    // 1 tạo mới giỏ hàng (chưa thanh toán), 0 đã thanh toán(đã thanh toán)
    trang_thai:{
        type: Number,
        default: 1
    }
},{
    timestamps: true
})
module.exports = new mongoose.model('gioHangs', GioHangsSchema)

File: AppBanNoiThat/app/src/main/java/com/example/appbannoithat/Model/GioHang.kt
package com.example.appbannoithat.Model

data class GioHang(
    val _id : String,
    val nguoi_dung_id : String,
    val trang_thai : Number
)

data class GioHangReq(
    val nguoi_dung_id : String,
    val trang_thai : Number
)